### ISSUE TITLE
Filter and Show Projects list by Community District

### ISSUE THREAD

**Issue by TylerMatteo** [NYCPlanning/ae-cp-map#15]
## Acceptance Criteria

- [ ] When user selects a borough and CD from the Filter by District menu, user is navigated to new `/community-districts/<boroughId>/<communityDistrictId>/capital-projects` route and sees the content panel with the paginated list of projects
- [ ] Projects on the map are filtered to only those that overlap with selected CD. For projects that partially fall outside the CD, all parts of the project should still be visible.
- [ ] Selected CD boundary is highlighted ([figma](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=670-5575&t=KGRjmeAKGkxSYIRL-0))
- [ ] List of projects is paginated, with pagination arguments stored as query params. See `Page` [schema](https://github.com/NYCPlanning/ae-zoning-api/blob/main/openapi/openapi.yaml#L1120) from the underlying API for reference.
- [ ] User can click on a project to navigate to the nested route for that project: `/community-districts/<boroughId>/<communityDistrictId>/capital-projects/<managingCode>/<capitalProjectId>`
- [ ] User can dismiss the project list detail panel to return to root `/` route

### Designs

- [desktop](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=1187-33249&t=GOfIYpfW8egMdy6G-0) and [mobile](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=1329-23419&t=GOfIYpfW8egMdy6G-0)

### Out of scope
* Data export for projects within the CD. Disregard "Export" button in figma designs for the purposes of this Issue.

**Comment by TangoYankee** [comment]
Blocked by #18 and nycplanning/ae-zoning-api#247

**Comment by TylerMatteo** [comment]
We can consider this blocked by #14 because work can begin first on that issue and much of the code will be reusable here. Also making the effort lower for this one because much of the functionality can be borrowed from 14.

**Comment by TylerMatteo** [comment]
@horatiorosa I'm going to unassign you and move this back to Blocked, as it's effectively blocked by the similar issue for City Council Districts. Once it's unblocked, we can re-evaluate who will pick it up.
